src/main/scala/nl.knaw.dans.easy.multideposit/Command.scala

```diff
@@ -42,7 +42,8 @@ object Command extends App with DebugEnhancedLogging {
       case ParserFailedException(_, _) |
            EmptyInstructionsFileException(_) |
            InvalidDatamanagerException(_) |
-           InvalidInputException(_, _) => // do nothing
+           InvalidInputException(_, _) |
+           CompositeException(_) => // do nothing
```

Comment:
Can I try to convince you one more time to deal with `CompositeException` as soon as possible, rather than catching them at the end of the application's run?

`CompositeException` was designed to capture multiple `Throwable`s while running `collectResults`. You can therefore view it as a container that contains a collection of `Throwable` instances. As we do in various other instances, we want to generate an error report out of this collection, to have a user friendly error message.

When we only do this in the root of the application, we usually have no idea where this `CompositeException` is coming from (in itself it doesn't express any semantics of where the error comes from (compared with other error naming such as `EmptyInstructionsFileException` or `InvalidDatamanagerException`)). **Therefore I would say it is best practice to deal with `CompositeException` as early as possible, preferably even immediately after calling `collectResults`.**

Now to be completely honest with you: I did not do this myself in 

https://github.com/DANS-KNAW/easy-split-multi-deposit/blob/e4fe098df0187f6b5293255f06e9aff121512e08/src/main/scala/nl.knaw.dans.easy.multideposit/SplitMultiDepositApp.scala#L54

So maybe you can fix that for me in this PR, since it touches on it.  Thereby you convert `CompositeException` to another type of exception (let's say `InvalidInputException`), which removes the need for you to add `CompositeException` here.

Reply:
Let's leave the refactoring of the error handling code for another day.